includes/admin/views/html-notice-regenerating-lookup-table.php

```diff
@@ -1,26 +1,33 @@
 <?php
 /**
- * Admin View: Notice - Regenerating thumbnails.
+ * Admin View: Notice - Regenerating product lookup table.
  *
  * @package WooCommerce/admin
  */
 
 defined( 'ABSPATH' ) || exit;
 
+$pending_actions_url = admin_url( 'admin.php?page=wc-status&tab=action-scheduler&s=wc_update_product_lookup_tables&status=pending' );
+$cron_disabled       = defined( 'DISABLE_WP_CRON' ) && DISABLE_WP_CRON;
 ?>
 <div id="message" class="updated woocommerce-message">
 	<p>
-		<strong><?php esc_html_e( 'WooCommerce is updating product data in the background.', 'woocommerce' ); ?></strong>&nbsp;
+		<strong><?php esc_html_e( 'WooCommerce is updating product data in the background.', 'woocommerce' ); ?></strong><br/>
 		<?php
-		echo wp_kses_post(
-			sprintf(
-				/* Translators: 1: Link, 2: link close. */
-				__( 'Some queries and reports may not show accurate results until this finishes. It will take a few minutes and this notice will disappear when complete.', 'woocommerce' ),
-				'<a href="' . esc_url( admin_url( 'admin.php?page=wc-status&tab=action-scheduler&s=wc_update_product_lookup_tables' ) ) . '">',
-				'</a>'
-			)
-		);
+		esc_html_e( 'Product display, sorting, and reports may not be accurate until this finishes. It will take a few minutes and this notice will disappear when complete.', 'woocommerce' );
+
+		if ( $cron_disabled ) {
+			echo '<br><span style="color:red;">' . esc_html__( 'Warning: WP CRON has been disabled on your install which may prevent this update from completing.', 'woocommerce' ) . '</span>';
+		}
 		?>
-		&nbsp;<a href="<?php echo esc_url( admin_url( 'admin.php?page=wc-status&tab=action-scheduler&s=wc_update_product_lookup_tables&status=pending' ) ); ?>"><?php esc_html_e( 'View progress &rarr;', 'woocommerce' ); ?></a>
+		&nbsp;<a href="<?php echo esc_url( $pending_actions_url ); ?>">
```

Comment:
This is just a minor detail, but tabs are being added to the beginning of the text value of the `<a>` tag. This is resulting in an extra underlined white space at the beginning of the link text.

Reply:
Resolved